Makefile

```diff
@@ -26,6 +26,7 @@ GO_BUILD_ARGS = \
     -X '$(REPO)/internal/version.KubernetesVersion=$(K8S_VERSION)' \
   " \
 
+GOLANGCI_LINT_VER = "1.30.0"
```

Comment:
Please add it on top with the file with the other vars.

Reply:
@camilamacedo86 it is very confusing to know what you mean here... this in the middle of a series of variables which appear to be grouped for varies reasons.  It does go with the groupings for images, images and archs below.  you specify "top" and it is hard to claim that this is part of the group of build vars... but maybe... I don't see those in any particular order... I'll position this after k8s_ver but I'll need more specific guidance is that doesn't work for ya.